packages/integration/src/gerrit/core.ts

```diff
@@ -70,6 +70,26 @@ export function parseGerritGitilesUrl(
   };
 }
 
+/**
+ * Create a Gerrit Gitiles url for that targets a specific path.
+ *
+ * @param config - A Gerrit provider config.
+ * @param project - The name of the git project
+ * @param branch - The branch we will target.
+ * @param filePath - The absolute file path.
+ * @public
+ */
+export function createGerritGitilesUrl(
```

Comment:
The URL generating function in the GitLab integration uses a `build*` prefix, I think that might make sense here as well right? Just to keep things a bit consistent

Reply:
Yes, done!